f/connectors/locusmap/README.md

```diff
@@ -0,0 +1,38 @@
+# Locus Map: Import Data
+
+This script imports data from a Locus Map export file into a database table. It reads a file containing spatial data, transforms the data into a structured format, and inserts it into a PostgreSQL database table. Additionally, it downloads any attachments associated with the spatial data and saves them to a specified directory. It then deletes the temporary files.
+
+Locus Map exports data as a CSV, KML, and GPX. If attachments are included, then the export will be as a ZIP file compressing the spatial data file together with a directory containing the attachment files. (It is also possible to export data as a DXF or Ov2 file, but these are not commonly used formats, so this script does not intend to support them.)
+
+
+> [!IMPORTANT]
+> This script assumes that the temporary CSV file and attachments directory should always be deleted after processing.
```

Comment:
I'm fine with this, but just another idea:

You could consider putting a checkbox "Delete after ingest?" on the Windmill form. And I'd claim that if you did that, then you would no longer have to refer to this as "temporary" i.e. no longer need to make assumptions about where this file lives or what is the user's plan with it.

That said ... it would probably never get used this way :-)

Reply:
I like this idea! Thanks for the suggestion.